venv/Lib/site-packages/PIL/ImageQt.py

```
@@ -0,0 +1,220 @@
+    return qRgba(r, g, b, a) & 0xFFFFFFFF
+
+
+def fromqimage(im: QImage | QPixmap) -> ImageFile.ImageFile:
+    """
+    :param im: QImage or PIL ImageQt object
+    """
+    buffer = QBuffer()
+    qt_openmode: object
+    if qt_version == "6":
+        try:
+            qt_openmode = getattr(QIODevice, "OpenModeFlag")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)